app/views/legislation/questions/show.html.erb

```diff
@@ -29,6 +29,7 @@
   <div class="row">
     <div class="small-12 medium-9 column">
       <h3 class="quiz-question"><%= @question.title %></h3>
```

Comment:
Since we're now allowing `<h2>` and `<h3>` tags in the description, it might be a good time to fix this broken `<h3>` tag :thinking:. Since its content is also in the `<title>` element, and in order to be consistent with other `show` actions in the public area, IMHO this tag should be `<h1>`. We might need to adjust the styles accordingly.

Then we can use `<h2>` in the "Share" heading and replace the broken `<h4>` tag above with something that isn't a heading tag :thinking:.

What do you think?